docs/topics/installing-intellij-idea-plugin.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * docs/intellij-idea-plugin-guide/master.adoc
+
+[id="intellij-idea-plugin-extension_{context}"]
+= Installing the {ProductShortName} extension for IntelliJ IDEA
+
+You can install the {ProductShortName} extension in the Ultimate and the Community Edition releases of IntelliJ IDEA.
+
+.Prerequisites
+include::snippet_jdk-hardware-mac-prerequisites.adoc[]
+
+.Procedure
+
+. In IntelliJ IDEA, click the *Plugins* icon on the Activity bar.
```

Comment:
Actually, it should be "Click *Plugins* on the Activity bar. I fixed it in the Installing PR, after I deleted this file from the PR, since it is part of this PR - and that file has been changed since I wrote this PR.